pyne/source_sampling.pyx

```diff
@@ -490,6 +484,20 @@ cdef class Sampler:
     pass
 
 
+def convert_e_bounds(e_bounds):
+    """
+    """
```

Comment:
Function needs a real docstring :)

Reply:
Also this is a Python function (since it isn't there is no return type) so I think that this implictly converts the vector into a Python list on return.

Reply:
Sorry, I forgot it.

Reply:
No problem, that is what review is for :smile: